srfi-1-reference.scm

```diff
@@ -1,10 +1,6 @@
-;;; SRFI-1 list-processing library			-*- Scheme -*-
```

Comment:
We can't remove Olin's license without his permission.  His license here isn't an MIT license, and it specifically forbids removing the license.  (The MIT license notice appears in the document, but not here.)

Reply:
Good catch!  I've contacted them to see if they'd agree to relicense this work as Expat (MIT).

Reply:
I haven't heard back from them yet, so in the meantime I've used a custom license (via the LicenseRef SPDX facility).

Reply:
I now believe that I was wrong here.  As someone on the list pointed out, almost every SRFI, including SRFI 1, was changed to use the MIT license years ago.  The authors were contacted and approved the change.  So this change was reasonable.  Sorry about that.

Reply:
Could you point to this message on the list, or better, to Olin's original message saying so, if it was public on the list? I could register that link in the commit message for reference.

Reply:
I've left the original copyright notice and marked the file with the MIT SPDX license identifier.